google/cloud/storage/internal/object_write_streambuf.cc

```diff
@@ -212,34 +209,35 @@ void ObjectWriteStreambuf::FlushRoundChunk(ConstBufferSequence buffers) {
     // next.  Replace it with a SessionError so next_expected_byte and
     // resumable_session_id can still be retrieved.
     last_status_ = std::move(response).status();
-  } else {
-    // Reset the internal buffer and copy any trailing bytes from `buffers` to
-    // it.
-    auto* pbeg = current_ios_buffer_.data();
-    setp(pbeg, pbeg + current_ios_buffer_.size());
-    PopFrontBytes(buffers, rounded_size);
-    for (auto const& b : buffers) {
-      std::copy(b.begin(), b.end(), pptr());
-      pbump(static_cast<int>(b.size()));
-    }
+    return;
```

Comment:
Ditto.

Reply:
Ditto++.  I am going to merge as-is.  If I misread your comments we can fix the code in a followup PR.